### PR TITLE
Add alias for the redirect factory

### PR DESCRIPTION
**Pull request by vvasiloi**
To use autowiring

### CODE REVIEW

File: src/Resources/config/services/factory.xml [hunk 1]
```diff
@@ -9,5 +9,6 @@
                  decorates="setono_sylius_redirect.factory.redirect">
             <argument type="service" id="setono_sylius_redirect.custom_factory.redirect.inner" />
         </service>
+        <service id="Setono\SyliusRedirectPlugin\Factory\RedirectFactoryInterface" alias="setono_sylius_redirect.custom_factory.redirect" />
```
Comment (loevgaard) [inline src/Resources/config/services/factory.xml:12]: It should not alias the custom factory should it?

Comment (vvasiloi) [reply src/Resources/config/services/factory.xml:12]: That's what Symfony suggested. I guess because that's the service implementing the interface.

Comment (loevgaard) [reply src/Resources/config/services/factory.xml:12]: I am not 100% sure, but I am 90% sure it should be `setono_sylius_redirect.factory.redirect`

Comment (loevgaard) [reply src/Resources/config/services/factory.xml:12]: I am also not sure whether it even matters :D

Comment (vvasiloi) [reply src/Resources/config/services/factory.xml:12]: I honestly don't care. The result will be the same one my screen.

**Pull request merged**